presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,68 @@
+================================
+File Based System Access Control
+================================
+
+A system access control plugin enforces authorization at a global level,
+before any connector level authorization. You can provide your own
+implementation of ``SystemAccessControl``, as explained in
+:doc:`/develop/system-access-control`. Presto offers a built-in file based
+system access control plugin that supports specifying access control rules in a
+file.
+
+In order to enable this plugin, you must add an ``etc/access-control.properties``
+file containing two required properties, ``access-control.name`` which must be
+equal to ``file``, and ``security.config-file`` which must be set to the
+location of the config file. For example, if a config file named ``rules.json``
+resides in ``etc``, add an ``etc/access-control.properties`` with the following
+contents:
+
+.. code-block:: none
+
+   access-control.name=file
+   security.config-file=etc/rules.json
+
+The config file consists of a list of access control rules. These rules are
+matched in the order specified in the file. The user is
+granted the privileges from the first matching rule. All regular expressions
+default to ``.*`` if not specified.
+
+This plugin currently only supports catalog access control rules. If you want
+to limit access on a system level in any other way, you must implement a custom
+SystemAccessControl plugin (see :doc:`/develop/system-access-control`).
+
+Catalog Rules
+-------------
+
+These rules govern which catalogs particular users can access. The user is
+granted access to a catalog based on the first matching rule. If no rule
+matches, the user is denied access to the catalog. Thus, in absence of any
+rules, access to all catalogs is denied for all users when this plugin is
+enabled. Each rule is composed of the following fields:
+
+* ``user`` (optional): regex to match against user name.
+
+* ``catalog`` (optional): regex to match against catalog name.
+
+* ``allowed`` (required): boolean indicating whether a user has access to the catalog
+
+For example, if you want to disallow access to the ``mysql`` catalog to the user
+``alice``, but allow all other access, you can use the following rules:
+
+.. code-block:: json
+
+    {
+      "catalogs": [
+        {
+          "user": "alice",
+          "catalog": "mysql",
+          "allow": false
+        },
+        {
+          "user": ".*",
```

Comment:
I'm changing your example a little -- alice and ali will be allowed for all, but bob won't be. That way you a) show regexes and b) show that the default is to deny